RPIPs/RPIP-bounty_standards.md

```diff
@@ -0,0 +1,119 @@
+
+**Bounty Hunter**  
+
+A person or group that works to deliver the work specified by a well defined bounty.
+
+**Bounty Contact**  
+
+An individual that has expressed willingness to act as a point of contact with bounty hunters for a given bounty. This may be the bounty proposal author, a member of the GMC, or another Rocket Pool community member. The goal of the bounty contacts is to support and liase with bounty hunters so that they are better able to deliver the work specified in the bounty definition. Bounty contacts are listed in bounty definitions. 
+
+### Bounty Lifecycle
+![image](../assets/rpip-bounty_standards/bounty-lifecycle.png)
+
+#### Statuses
+**Draft - Proposed** - The bounty has been described and proposed to the GMC for adoption. The bounty is not yet eligible for adoption.  
+**Draft - Defined** - The bounty has been well defined and made ready for completion by a bounty hunter with the relevant skillsets if adopted by the GMC. The bounty is now eligible for adoption.  
+**Draft - Rejected** - The bounty has been rejected by the GMC at this time. The bounty is not eligible for completion.  
+**Open** - The bounty has been adopted by the GMC for completion and listed on relevant frontends. The bounty is now eligible for completion.  
+**Closed - Completed** - The bounty has been fully completed and paid out by the GMC. The bounty is no longer eligible for completion.  
+**Closed - Expired** - The bounty has met its expiration date. The bounty is no longer eligible for completion.  
+**Closed - Withdrawn** - The bounty has been withdrawn by the GMC. The bounty is no longer eligible for completion.  
+
+### Bounty Incentive Levers
+Bounty Incentive Levers are OPTIONAL incentives that the pDAO empowers the GMC to use to incentivise the generation and successful completion of bounty proposals.
+
+#### Lever Incentives
+* The incentive attached to each lever MAY be set by the GMC. 
+* Levers MAY have an absolute (fixed amount) component and/or a relative (percentage of total bounty payout) component. 
+* The payout of incentives for a given bounty MUST NOT reduce the bounty completion payout. 
+* When incentive levers are modified by the GMC, incentives SHOULD be paid out in such a way that ambiguity favours incentive recipients.
+* The total cost of lever incentives is not fixed. The GMC SHOULD set lever incentives with caution when funds are limited. 
+
+#### Payout
+* Bounty incentives SHOULD be paid out of the GMC budget.
+* Incentive payouts SHOULD take place within the GMC's regular payout schedule.
+* Incentive payouts MUST be denominated in dollars, and MUST be paid out in one of the GMC's accepted payout token options selected by the recipient.
+
+#### Lever A: Proposal Incentive
+The proposal incentive lever aims to influence the number of bounty proposals that are recieved by the GMC.
+* This incentive MUST be paid to the proposal author for a bounty proposal that is adopted by the GMC. 
```

Comment:
Should there be a discretionary provision for similar bounties? Thinking an extreme here where the GMC combines two draft proposals, or otherwise elevates a bounty proposal with an idea from another party - would probably want to explicitly state they can award the other party.

Reply:
Changed to SHOULD to avoid the complexity.